sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -180,17 +180,19 @@ def create_table(
             self,
             **kwargs  # type: Any
     ):
-        # type: (...) -> TableItem
+        # type: (...) -> Dict[str,str]
         """Creates a new table under the current account.
 
-        :return: TableItem created
-        :rtype: TableItem
+        :return: Dictionary of response headers from service
+        :rtype: dict[str,str]
         :raises: ~azure.core.exceptions.HttpResponseError
         """
         table_properties = TableProperties(table_name=self.table_name, **kwargs)
         try:
-            table = self._client.table.create(table_properties)
-            return TableItem(table=table)
+            metadata, _ = self._client.table.create(
```

Comment:
What's in this metadata? Do we need to clean out unnecessary information?

Reply:
The returned metadata has these keys: `client_request_id`, `request_id`, `version`, `date`, and `etag` always. On `create_entity` the service also returns `preference_applied` and `content_type`. `client_request_id`, `request_id`, and `preference_applied` are all `None` in the tests